script/.contributors_aliases.json

```diff
@@ -189,6 +189,10 @@
     "mails": ["anjsimmo@gmail.com", "a.simmons@deakin.edu.au"],
     "name": "Andrew J. Simmons"
   },
+  "antonioglez-23@hotmail.com": {
+    "mails": ["antonioglez-23@hotmail.com"],
+    "name": "Antonio Glez 23"
```

Comment:
Not sure about the name here but there's two "Antonio", we need to distinguish them

Reply:
Who is the other one? (It's not [this](https://github.com/pylint-dev/astroid/commit/0361eb460fc92da6ac3c69637cc6a8698b509d88) one?)

Reply:
I need to look in detail the link between github account and email to be sure (not doable with only the gît log)

Reply:
Commit come from #8721 and #8728, I'm going to fix this accordingly